docs/source/instrument/_install_new_instrument.rst

```diff
@@ -27,7 +27,7 @@ to the documentation for each step.
 #. Assign a MongoDB catalog for databroker: `instructions for BCDA <https://git.aps.anl.gov/bcda/bluesky-catalogs/-/blob/master/README.md>`__
 #. `databroker catalog configuration <https://bcda-aps.github.io/bluesky_training/instrument/_configure_databroker.html#setup-your-databroker-catalog-configuration>`__
 #. `initial configuration of instrument <https://bcda-aps.github.io/bluesky_training/instrument/_configure_bluesky_instrument.html#initial-configuration>`__
-#. `start a GitHub repository <https://bcda-aps.github.io/bluesky_training/reference/_github_create_repo.html>`__
+#. `start a remote repository <https://bcda-aps.github.io/bluesky_training/instrument/_install_new_instrument.html#start-version-control>`__
```

Comment:
This link could be a local reference since it is to a section in this same .rst file.  The section needs to have a label preceding it first.

Reply:
See PR #220